README.md

```diff
@@ -262,7 +262,7 @@ The native notifications API is hosted at `nativepush-cluster1.pusher.com` and o
 
 You can send pushes by using the `notify` method, either globally or on the instance. The method takes two parameters:
 
-- `interests`: An Array of strings which represents the interests your devices are subscribed to. These are akin to channels in the DDN with less of an epehemeral nature. Note that currently, you can only send to _one_ interest.
+- `interests`: An Array of strings which represents the interests your devices are subscribed to. These are akin to channels in the DDN with less of an epehemeral nature. Note that currently, you can only publish to, at most, _ten_ interests.
```

Comment:
I think it would be better to keep the limits in a central table that we can refer from the docs but YMMV